tests/testthat/_snaps/size.md

```diff
@@ -61,22 +61,31 @@
       (expect_error(vec_size_common(.size = "foo")))
     Output
       <error/rlang_error>
-      Error in `vec_size_common()`:
+      Error:
       ! `.size` must be a single number, not a string.
     Code
       (expect_error(vec_size_common(.size = 1:2)))
     Output
       <error/rlang_error>
-      Error in `vec_size_common()`:
+      Error:
```

Comment:
I think this was expected right? It no longer reports itself because of that `.call = caller_env()` argument?

Reply:
well this is a case of internal error.

Thanks for being patient with me today :D